flash_notebooks/finetuning/image_classification.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
<p>Create the <code>ImageClassifier</code>  task. By default, the ImageClassifier task uses a resnet-18 backbone to train or finetune your model.</p><p> </p><p>quick question: don't you need to specify num_classes in the datamodule?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
It is inferred automatically.